### ISSUE TITLE
AMA intake | Create tracked item for HLR informal conference requests

### ISSUE THREAD

**Issue by abbyraskinUSDS** [department-of-veterans-affairs/caseflow#5280]
When a user selects that a veteran requested an informal conference. We should be creating a tracked item in BGS/VBMS to keep track of that.
To do this, we must first create a claimant letter in BGS. It would be great to get more information around where we get the letter (Is it part of the intake process?). Once the claimant letter is generated, we can generate a tracked item.

<img width="923" alt="screen shot 2018-07-24 at 1 49 35 pm" src="https://user-images.githubusercontent.com/5531789/43165319-744ec90c-8f48-11e8-970d-8a58505f1f34.png">

<img width="937" alt="screen shot 2018-07-24 at 1 50 29 pm" src="https://user-images.githubusercontent.com/5531789/43165356-918a5522-8f48-11e8-89e1-ab89bd2ca7bf.png">

Full documents in the DocumentServiceInputOutput

## Acceptance Critieria
- When processing a HLR in AMA Intake, when the request for informal conference option is selected and the intake is submitted, create an informal conference tracked item on the resulting EPs.
- If there is both a rating & non-rating EP associated with the HLR, there should be an informal conference tracked item on both EPs

## Technical notes
- Code for Ruby-bgs repo
  - Add code to ruby_bgs to make a call to manageClaimantLetter_V2 to create the claimant letter
  - Add code to ruby_bgs to create the "Informal Hearing" tracked item, using generateTrackedItems
- In Caseflow repo code, after establish_claim:
  - Write a method that makes the bgs calls above, after establish_claim
  - Create database fields to track when these requests have succeeded
- Open questions
  - Do we want "followUp" to be true or false?
  - Do we need to have a letter as an input to manageClaimantLetter_V2? If so, where do we get it from?
  - How does the tracked item get connected to the letter?
  - How do we indicate in the tracked item that the item is a request for an informal_conference

**Comment by abbyraskinUSDS** [comment]
@shanear to investigate

**Comment by abbyraskinUSDS** [comment]
- We should create 2 separate tracked items if we create both a non-rated and rated EP.

**Comment by abbyraskinUSDS** [comment]
I updated the AC @shanear

**Comment by kahlouie** [comment]
Kara to add more info on this.

**Comment by leikkisa** [comment]
Adding blocked because after reviewing the WSDL we feel that we need to set up another meeting with BGS to determine what needs to be done for this.

**Comment by leikkisa** [comment]
Waiting on response from Anil

**Comment by leikkisa** [comment]
We think we have what we need, so I am closing this in favor of the development tickets:
https://github.com/department-of-veterans-affairs/ruby-bgs/issues/65
https://github.com/department-of-veterans-affairs/caseflow/issues/7117
https://github.com/department-of-veterans-affairs/caseflow/issues/7118
